packages/dds/tree/src/feature-libraries/forest-summary/forestSummarizer.ts

```diff
@@ -90,32 +116,51 @@ export class ForestSummarizer implements Summarizable {
 			);
 			fieldMap.set(key, innerCursor as ITreeCursorSynchronous & ITreeSubscriptionCursor);
 		});
-		const encoded = this.codec.encode(fieldMap, this.encoderContext);
 
+		const forestSummaryBuilder = new SummaryTreeBuilder();
+		// Let the incremental summary builder know that we are starting a new summary. It returns whether
+		// incremental encoding is enabled.
+		const shouldEncodeIncrementally = this.incrementalSummaryBuilder.startingSummary(
```

Comment:
This will be disabled before merging this PR. A follow up PR will gate this behind a config and / or behind format version of FieldBatchCodec.

Disabling this will make the change backwards compatible where the summary generated will be the same as before this change.

Reply:
If you want to work toward getting this PR merged, you should fix this. If you code can't be tested and reviewed without this, then you need to fix that before you can proceed with merging as we can't merge things that aren't testable and readable.